mula/scheduler/server/serializers/schedule.py

```diff
@@ -0,0 +1,17 @@
+from datetime import datetime
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class Schedule(BaseModel):
+    model_config = ConfigDict(from_attributes=True)
+
+    hash: str | None = Field(None, max_length=32)
+
+    data: dict | None = {}
```

Comment:
I think using a `Field` with a default factory for dict is safer. Also, if the field is not supposed to be nullable, but does have a default value, then the `None` type should be removed